xml/yast2_ncurses.xml

```diff
@@ -285,7 +285,7 @@
   </variablelist>
  </sect1>
 
- <sect1 xml:id="sec.yast.ncurses_tasten">
+ <sect1 xml:id="sec.yast.text.restricted_keys">
   <title>Restriction of Key Combinations</title>
```

Comment:
i don't like the word "restrict" in this context (neither do i like "occupied"). "overloaded" would be nice but is probably too technical.
how about sec.yast.ncurses.key_conflict and "Key Combinations Conflicts With Window Managers"